### ISSUE TITLE
TestGCQueueTransactionTable failed during testrace

### ISSUE THREAD

**Issue by BramGruneir** [cockroachdb/cockroach#16642]
https://teamcity.cockroachdb.com/viewLog.html?buildId=276982&
```
<autogenerated>:14: storage/gc_queue_test.go:740, condition failed to evaluate within 45s: g: expected status COMMITTED, but found PENDING
------- Stdout: -------
I170620 20:04:30.001888 661 storage/replica_command.go:178  [s1,r1/1:/M{in-ax}] test injecting error: boom```

**Comment by tamird** [comment]
Hit this twice in a row. Looking.

**Comment by tamird** [comment]
Looks like this was introduced by #16632. Assigning to @bdarnell.

**Comment by tschottdorf** [comment]
Fixed in #16695.
